### PR TITLE
Andrewbladon/open map url data update

### PR DESCRIPTION
**Pull request by AndrewBladon**
This PR changes two of the web map portal IDs in the sample code and updates the readme file. There are two new web map IDs on top of the existing geological map layer: one for showing terrestrial ecosystems of the world, and the other for recent hurricanes, cyclones and typhoons.

Could you review these changes please @tanneryould and @lsmallwood?

### CODE REVIEW

File: ArcGISRuntimeSDKQt_CppSamples/Maps/OpenMapUrl/README.md [hunk 1]
```diff
@@ -20,12 +20,14 @@ A web map can be selected from the drop-down list. On selection the web map disp
 
 ## Relevant API
 
-* Map
+* ArcGISMap
```
Comment (tanneryould) [inline ArcGISRuntimeSDKQt_CppSamples/Maps/OpenMapUrl/README.md:23]: `ArcGISMap` is the name of the class in a different API (Java, I believe). Our equivalent is `Map`.
```suggestion
* Map
```

File: ArcGISRuntimeSDKQt_QMLSamples/Maps/OpenMapUrl/README.md [hunk 2]
```diff
@@ -20,12 +20,14 @@ A web map can be selected from the drop-down list. On selection the web map disp
 
 ## Relevant API
 
-* Map
+* ArcGISMap
```
Comment (tanneryould) [inline ArcGISRuntimeSDKQt_QMLSamples/Maps/OpenMapUrl/README.md:23]: Same here regarding our class name vs other SDKs.
```suggestion
* Map
```

File: ArcGISRuntimeSDKQt_CppSamples/Maps/OpenMapUrl/OpenMapUrl.qrc [hunk 3]
```diff
@@ -7,8 +7,9 @@
         <file>OpenMapUrl.cpp</file>
         <file>README.md</file>
         <file>geology.jpg</file>
-        <file>population.png</file>
         <file>SwitchMap.png</file>
-        <file>Tapestry.png</file>
+        <file>ecosystems.png</file>
+        <file>traces.png</file>
     </qresource>
+    <qresource prefix="/"/>
```
Comment (tanneryould) [inline ArcGISRuntimeSDKQt_CppSamples/Maps/OpenMapUrl/OpenMapUrl.qrc:14]:
```suggestion
```

**Review by tanneryould — CHANGES_REQUESTED** [review]
Couple of minor changes but otherwise looks good

**Review by lsmallwood — APPROVED** [review]
LGTM - nothing to add

**Review by tanneryould — APPROVED** [review]
Code suggestions applied - LGTM

**Pull request merged**